src/azul/__init__.py

```diff
@@ -1586,6 +1586,26 @@ def vpc_cidr(self) -> str:
     def vpn_subnet(self) -> str:
         return self.environ['azul_vpn_subnet']
 
+    def content_security_policy(self, nonce: Optional[str] = None) -> str:
```

Comment:
I don't think this is a good place for this method. Can this not live in `azul.chalice`, ideally as a method of AzulChaliceApp?